tests/RobotFramework/tests/mqtt/custom_sub_topics_tedge-mapper-aws.robot

```diff
@@ -0,0 +1,29 @@
+*** Settings ***
+Documentation       Purpose of this test is to verify that tedge-mapper-aws subscribes the topics that are configured as aws.topics
+
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO
+
+Suite Setup         Custom Setup
+Suite Teardown      Get Logs
+
+Test Tags           theme:mqtt    theme:aws
+
+
+*** Test Cases ***
+Publish events to subscribed topic
+    Execute Command    mosquitto_pub -t tedge/events/event-type -m '{"text": "Event"}'
+    Should Have MQTT Messages    aws/td/events/event-type
+
+Publish measurements to unsubscribed topic
+    Execute Command    mosquitto_pub -t tedge/measurements -m '{"temperature": 10}'
+    Sleep    5s    reason=If a message is not published in 5s, it will never be published.
+    Should Have MQTT Messages    aws/td/measurements    minimum=0    maximum=0
+
+
+*** Keywords ***
+Custom Setup
+    Setup
+    Execute Command    sudo tedge config set aws.topics "tedge/events/+"
+    Execute Command    sudo systemctl start tedge-mapper-aws.service
+    Sleep    1s    reason=It fails without this. It needs to wait until tedge-mapper-aws starts
```

Comment:
It would be better to check the health message related to the tedge-mapper-aws service, for example:

```suggestion
    ThinEdgeIO.Service Health Status Should Be Up    tedge-mapper-aws
```